clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -1582,3 +1580,17 @@ def remove_tmp_dmc_folder(bids_dir, image_id):
     tmp_dcm_folder_path = join(bids_dir, f"tmp_dcm_folder_{str(image_id).strip(' ')}")
     if exists(tmp_dcm_folder_path):
         rmtree(tmp_dcm_folder_path)
+
+
+def load_clinical_csv(clinical_dir: str, filename: str) -> pd.DataFrame:
+    import re
+    from pathlib import Path
+
+    pattern = filename + "(_\d{1,2}[A-Za-z]{3}\d{4})?.csv"
```

Comment:
Are we expecting to have exactly one file matching this pattern ?

If yes, I think we should verify this and raise if more than one file was found. Currently the function returns the last one found. Maybe something like this:

```python
files_matching_pattern = [
    f for f in Path(clinical_dir).rglob("*.csv") if re.search(pattern, (z.name))
]
if len(files_matching_pattern) != 1:
    raise IOError(
        f"Expecting to find exactly one file in folder {clinical_dir} "
        f"matching pattern {pattern}. {len(files_matching_pattern)} "
        f"files were found instead : {'\n- '.join(files_matching_pattern)}"
    )
try:
    return ....
```

Reply:
Unless you download the data twice, yes, we expect only one file.

Reply:
Done !